### DISCUSSION TITLE
flask app timeout urgent issue

### DISCUSSION THREAD

**Discussion by themeswordpress** [pallets/flask#4426] [Q&A]
Dears,

Hope all is well.

I have an application, my app build based on the flask, my issue is as follows, 

after sending 1000 to 3000 requests per minute to my flask app the flask going down (no response ) kindly keep in your mind that my issue is not a resource issue because my memory percentage is less than 20% and CPU 30 to40 % 

so how can I fix this issue in flask 

PS: I did the same testing in fastapi and there is no issue with them

**Comment by ThiefMaster** [comment]
Not sure what you expect here. You didn't even mention how you are running the app. gunicorn? uwsgi? `flask run`? Also, this is not the right place for issues with your own code that are not necessarily flask problems. I'm moving this to discussions.

**Comment by themeswordpress** [comment]
flask run , but same issue with uwsgi and gunicorn
